<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logon</class>
 <widget class="QWidget" name="Logon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DesignerTest/logon.qrc">
    <normaloff>:/app.ico</normaloff>:/app.ico</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditId"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPW">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Password:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditPW</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPW">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelId">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;id:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditId</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>88</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonOk">
      <property name="text">
       <string>Ok</string>
      </property>
      <property name="icon">
       <iconset resource="../DesignerTest/logon.qrc">
        <normaloff>:/attack_minion.png</normaloff>:/attack_minion.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditId</tabstop>
  <tabstop>lineEditPW</tabstop>
  <tabstop>ButtonOk</tabstop>
 </tabstops>
 <resources>
  <include location="../DesignerTest/logon.qrc"/>
 </resources>
 <connections/>
</ui>
